distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1008,13 +1009,15 @@ public void testGetMetacardResourceStatusCached()
                 DEFAULT_URL_RESOURCE_READER_ROOT_RESOURCE_DIRS, productDirectory});
 
         String url = REST_PATH.getUrl() + "sources/ddf.distribution/" + metacardId;
-        String resourceCachedXpath = String.format("//*[@name='%s']/*/text()", Metacard.RESOURCE_CACHE_STATUS);
+        String resourceCachedXpath = String.format("//*[@name='%s']/*/text()",
+                Metacard.RESOURCE_CACHE_STATUS);
 
         get(url + "?transform=resource").then();
 
         get(url).
-            then().
-                body(hasXPath(resourceCachedXpath, is("true")));
+                then()
+                .
```

Comment:
Not sure if it's just how github is showing it, but this looks really weirdly formatted.